LottieSample/src/main/kotlin/com/airbnb/lottie/samples/WarningsDialogFragment.kt

```diff
@@ -38,14 +38,13 @@ class WarningsDialogFragment : DialogFragment() {
     }
 
     companion object {
-        private val ARG_WARNINGS = "warnings"
-
-        internal fun newInstance(warnings: ArrayList<String>): WarningsDialogFragment {
-            val args = Bundle()
-            args.putStringArrayList(ARG_WARNINGS, warnings)
-            val frag = WarningsDialogFragment()
-            frag.arguments = args
-            return frag
+
+        private const val ARG_WARNINGS = "warnings"
+
+        internal fun newInstance(warnings: ArrayList<String>) = WarningsDialogFragment().apply {
```

Comment:
make the code more concise.
reference: 
[http://kotlinlang.org/docs/reference/functions.html#single-expression-functions](http://kotlinlang.org/docs/reference/functions.html#single-expression-functions)
[http://kotlinlang.org/api/latest/jvm/stdlib/kotlin/apply.html](http://kotlinlang.org/api/latest/jvm/stdlib/kotlin/apply.html)